doc/source/conduce-entities.rst

```diff
@@ -0,0 +1,159 @@
+
+-----------------
+Creating entities
+-----------------
+
+An entity is a series of samples that describe the state of and object over a period of time.  A sample is the representation of an entity's momentary state.  Samples are created from user data.  The data should describe people or things that exist in some location during some time span.  Data can be ingested from static files with thousands of records or real-time streams that provide state updates in real-time.  As an example we'll ingest the following real-time shipment data into an existing dataset named ``shipments``:
+
+.. list-table:: Source data: real-time shipment status
+   :header-rows: 1
+   :widths: auto
+
+   * - ID
+     - Method
+     - Date
+     - Latitude
+     - Longitude
+     - Value
+     - Status 
+   * - 1
+     - ground
+     - 2017-10-24T10:22:09+00:00
+     - 37.952861
+     - -91.922607
+     - $102,325.26
+     - transit
+   * - 2
+     - air
+     - 2017-10-24T10:22:11+00:00
+     - 38.289937
+     - -100.107422
+     - $3,216,757.84
+     - transit
+   * - 3
+     - ground
+     - 2017-10-24T10:22:08+00:00
+     - 40.354917
+     - -89.104614
+     - $204,694.41
+     - processing
+   * - 4
+     - air
+     - 2017-10-24T10:22:06+00:00
+     - 35.85344
+     - -115.993652
+     - $102,325.26
+     - transit
+   * - 5
+     - freight
+     - 2017-10-24T10:22:18+00:00
+     - 25.029996
+     - -134.318848
+     - $1,431,873,345.01
+     - transit
+
+The data in this table represents the present location of 5 shipments.  Each shipment has an ID, a shipping method, a date when it was last updated and a location in geographic coordinates.  Additionally the current state and value of the shipment is documented.  A Conduce entity is able to capture and represent all of this information.  These data records will be ingested as the first samples of 5 unique entities of the ``shipments`` dataset.
+
+An entity sample requires four fields:
+ **id**
+     a unique string that represents a specific object that exists.
+ **kind**
+     the type or category of the object.
+ **time**
+     A :py:class:`datetime.datetime` at which the entity state described by this sample is valid.
+
+And one of:
+ **point**
+     A coordinate pair that defines a location in a 2D cartisian coordinate system.
+ **path**
+     A list of coordinates that define a series of connected line segments in a 2D cartisian coordinate system.
+ **polygon**
+     A list of coordinates that define a closed shape in a 2D cartisian coordinate system.
+
+Each can be specified in ``(x,y)`` or ``(latitude,longitude)``. If coordinate types are mixed or multiple coordinate types are specified an error will result.  If more than one location is defined, only one will be used.  Precidence is undefined.
```

Comment:
super nitpick: This might be a little confusing since x,y & lat,lon have special formatting & look like they could translate easily to code.  So it sort of looks a user can use tuples.  Also, x, y are real key values, but `latitude` & `longitude` are not.

Reply:
True.